test/player_integration.js

```diff
@@ -433,6 +433,15 @@ describe('Player', () => {
       expect(video.playbackRate).toBe(1);
     });
 
+    it('in sequence mode', async () => {
```

Comment:
This only validates that the content loads.  For sequence mode, I think it would be better to validate:

1. It doesn't hang if you seek
2. It plays to the end of the stream

Can you seek right after load, and validate that it makes it to an ended state?  Since the fake stream is 30 seconds long, maybe seek to 20?

Everything looks good otherwise.